tests/performance_tests/performance_test_basic.py

```diff
@@ -21,26 +21,144 @@
 relaunchTimeout = 30
 emptyBlockGoal = 5
 
-def waitForEmptyBlocks(node1):
+class blockData():
+    def __init__(self):
+        self.transactions = 0
+        self.cpu = 0
+        self.net = 0
+        self.elapsed = 0
+        self.time = 0
+        self.latency = 0
+
+class chainData():
+    def __init__(self):
+        self.blockLog = []
+    def sum_transactions(self):
+        total = 0
+        for block in self.blockLog:
+            total += block.transactions
+        return total
```

Comment:
Instead of having this always dynamically calculated, should there be internal variables that track and store these data points which are updated each time a `blockData` is added to the `blockLog`?  This might be premature. Just worth thinking about.

Reply:
Definitely do this and eliminate all the boilerplate methods.